### ISSUE TITLE
Modded Namelists appear "broken"

### ISSUE THREAD

**Issue by MrFunEGUY** [eliasdoehne/stellaris-dashboard#100]
Picture shows what I mean. Some are not broken, per se, but they are showing the modded name localisation key instead of the name itself. This is from the mod "[Just Name Lists](https://steamcommunity.com/sharedfiles/filedetails/?id=2825897107)", using the HUM3 namelist, as you can tell. Additionally, the names of other nations appear broken if they use any modded names. I don't know why the one nation has "AoB," because that feels like a 3.6 problem that you already fixed. Everything looks fine for these nations in game.

![Image 019](https://user-images.githubusercontent.com/81896235/212572385-cff1c3a8-17e8-4569-9bee-bc0702681bbb.png)
![Image 069](https://user-images.githubusercontent.com/81896235/212735858-f7201f89-a48f-4949-aa37-2ea97ca33725.png)

**Comment by eliasdoehne** [comment]
Interesting, when using mods that change the name lists, some manual work is necessary, [there are some instructions here](https://github.com/eliasdoehne/stellaris-dashboard/blob/master/README.md#names-and-localizations).

If you followed that process and still get this output, could you please upload an example save file and the database file that produced this screenshot? (you may need to zip it before uploading)

**Comment by MrFunEGUY** [comment]
> Hello @MrFunEGUY , when using mods that change the name lists, some manual work is necessary, [there are some instructions here](https://github.com/eliasdoehne/stellaris-dashboard/blob/master/README.md#names-and-localizations).
> 
> If you followed that process and still get this output, could you please upload an example save file and the database file that produced this screenshot? (you may need to zip it before uploading)
[Name Issue.zip](https://github.com/eliasdoehne/stellaris-dashboard/files/10430047/Name.Issue.zip)


Ok great, that's fixed., thanks a lot!

However, the "AoB" name issue still appears for this one empire:
![Image 078](https://user-images.githubusercontent.com/81896235/212781826-bfc3b4da-27e7-4566-bd67-a166ed4fac13.png)

[Name Issue.zip](https://github.com/eliasdoehne/stellaris-dashboard/files/10430047/Name.Issue.zip)

**Comment by chennin** [comment]
Hi! I noticed the same/related problem that I'm looking at:

![image](https://user-images.githubusercontent.com/2653277/215587243-c792da14-0fab-4b3e-828c-d7262941dad2.png)

However, I can't replicate your case using your save:

![image](https://user-images.githubusercontent.com/2653277/215587363-42dedd2f-fcc1-48f4-b7e3-16f81eb86851.png)

In your `localisation/english/name_system_l_english.yml` (the copy the dashboard can access), do you have an AofB line? It looks like this for me (3.6.1):

```
AofB:0 "$1$ of $2$" # Used in empire names like 'Empire of Sol'.
```

If not, what version of Stellaris are you running?

Also, where is "Seurcaen" from? It doesn't seem to be in Just Name Lists 3.4?

**Comment by MrFunEGUY** [comment]
I didn't know I needed that file. Is there a list of all the vanilla files that are needed if you move the localisation to a different folder in order to accommodate mods? I have moved that file and reparsed the save, and the names are nearly all correct now which is simply amazing. Thanks for that.

"Seurcaen" is from [Random Species Names - Updated](https://steamcommunity.com/sharedfiles/filedetails/?id=2117708790).

**Comment by chennin** [comment]
Not sure. All of them? :)

So far I just point my dashboard at the game's default localisation folder, but I don't use mods that add names like the above (yet).  If I did I might find some way to copy all `localisation/english/*.yml` files in to one folder, and point the dashboard at that.

**Comment by MichaelMakesGames** [comment]
This was closed by #116
